db/migrate/20190320144534_create_contact.rb

```diff
@@ -0,0 +1,15 @@
+class CreateContact < ActiveRecord::Migration[5.2]
+  def change
+    create_table :contact do |t|
+      t.integer :provider_id, null: false
+      t.text :type
```

Comment:
Should all of these columns be nullable?

Reply:
Done. Thanks for the feedback.